### PR TITLE
Update motion_submitter create action.

### PR DESCRIPTION
**Pull request by reiterl**
Add an ids unique check.
Add a test.

### CODE REVIEW

File: openslides_backend/action/motion_submitter/create.py [hunk 1]
```diff
@@ -36,4 +37,15 @@ def update_instance(self, instance: Dict[str, Any]) -> Dict[str, Any]:
...
+
+        # check, if (user_id, motion_id) already in the databse.
+        filter = And(
+            FilterOperator("user_id", "=", instance["user_id"]),
+            FilterOperator("motion_id", "=", instance["motion_id"]),
+        )
+        db_instances = self.database.filter(
+            collection=self.model.collection, filter=filter, mapped_fields=["user_id"],
+        )
+        if len(db_instances) > 0:
```
Comment (jsangmeister) [inline openslides_backend/action/motion_submitter/create.py:49]: you can uses the `exists(...)` function of the datastore for this

**Review by jsangmeister — CHANGES_REQUESTED** [review]

File: openslides_backend/action/motion_submitter/create.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/motion_submitter/create.py]: Okay, updated.

**Review by jsangmeister — APPROVED** [review]
You can squash and merge

**Comment by reiterl** [comment]
Rebased + Squashed.

**Pull request merged**